templates/tailwind/block/pricing-section/configuration/pricing-section-schema.json

```diff
@@ -0,0 +1,197 @@
+                          "position": 1
+                        },
+                        "month": {
+                          "label": "Month",
+                          "default": true,
+                          "position": 0
+                        }
+                      }
+                    },
+                    "label": "Frequency",
+                    "private": false,
+                    "optional": false,
+                    "position": 1,
+                    "description": "The payment frequency."
+                  }
+                }
+              },
+              "label": "Price",
+              "private": false,
+              "optional": false,
+              "position": 3,
+              "description": "The plan price."
+            },
+            "description": {
+              "type": {
+                "type": "text",
+                "minimumLength": 1
+              },
+              "label": "Description",
+              "private": false,
+              "optional": true,
+              "position": 4,
+              "description": "Plan description"
+            },
+            "features": {
+              "type": {
+                "type": "list",
+                "items": {
+                  "type": "text",
+                  "minimumLength": 1
+                },
+                "maximumLength": 6,
+                "minimumLength": 3
+              },
+              "label": "Features",
+              "private": false,
+              "optional": false,
+              "position": 5,
+              "description": "The plan features."
+            },
+            "cta": {
+              "type": {
+                "type": "structure",
+                "attributes": {
+                  "label": {
+                    "type": {
+                      "type": "text",
+                      "minimumLength": 1
+                    },
+                    "label": "Label",
+                    "private": false,
+                    "optional": false,
+                    "position": 0,
+                    "description": "The CTA label."
+                  },
+                  "link": {
+                    "type": {
+                      "type": "text",
+                      "minimumLength": 1
+                    },
+                    "label": "Link",
+                    "private": false,
+                    "optional": false,
+                    "position": 1,
+                    "description": "The CTA link."
+                  }
+                }
+              },
+              "label": "CTA",
+              "private": false,
+              "optional": false,
+              "position": 6,
+              "description": "The call to action button."
+            }
+          }
+        },
+        "itemLabel": "Plan"
+      },
+      "label": "Plans",
+      "private": false,
+      "optional": false,
+      "position": 3,
+      "description": "The plans list."
```

Comment:
In this case, I suggest modeling the company without the highlight option and using a fixed structure with two keys (first and second). If we use a list and the highlight feature, things get messy when we switch the cards order.

<img width="930" alt="image" src="https://github.com/user-attachments/assets/9fc1fc88-be57-4d33-b9c7-4799e0d09729" />